### ISSUE TITLE
Returned color is always 100% transparent

### ISSUE THREAD

**Issue by pronebird** [sgl0v/MSColorPicker#1]
Demo works though.

``` objc
- (IBAction)changeColor:(id)sender {
    MSColorSelectionViewController *controller = [[MSColorSelectionViewController alloc] init];
    UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:controller];

    UIBarButtonItem *doneBtn = [[UIBarButtonItem alloc] initWithTitle:NSLocalizedString(@"Done", ) style:UIBarButtonItemStyleDone target:self action:@selector(dismissColorPicker:)];
    controller.navigationItem.rightBarButtonItem = doneBtn;

    controller.delegate = self;
    controller.color = self.view.tintColor;

    [self presentViewController:navigationController animated:YES completion:nil];
}

- (void)dismissColorPicker:(id)sender {
    [self dismissViewControllerAnimated:YES completion:nil];
}

- (void)colorViewController:(MSColorSelectionViewController *)colorViewCntroller didChangeColor:(UIColor *)color {
    self.view.window.tintColor = color;
    colorViewCntroller.view.window.tintColor = color;
}
```

**Comment by pronebird** [comment]
If you look at the `setColor` setter, it proxies color value to subview which is not created yet. Your demo works because you force viewDidLoad by using .view property of color selection controller. 

It would make sense to retain the initial color and pass it to subview in `viewDidLoad`.

**Comment by sgl0v** [comment]
Fixed this issue in version 0.2.1 . In the end I decided to force a view creation in `setColor` method.
